pipeline/config-prod.groovy

```diff
@@ -341,33 +341,33 @@ environments {
             }
             resources {
                 node {
-                    cpu_request = "100m"
-                    cpu_limit = "150m"
-                    memory_request = "512Mi"
-                    memory_limit = "1Gi"
+                    cpu_request = "20m"
+                    cpu_limit = "100m"
+                    memory_request = "160Mi"
+                    memory_limit = "256Mi"
                     replica_min = 2
                     replica_max = 4
                 }
                 nginx {
-                    cpu_request = "100m"
-                    cpu_limit = "150m"
-                    memory_request = "256Mi"
-                    memory_limit = "512Mi"
-                    replica_min = 2
-                    replica_max = 4
+                    cpu_request = "10m"
+                    cpu_limit = "50m"
+                    memory_request = "96Mi"
+                    memory_limit = "160Mi"
+                    replica_min = 3
```

Comment:
Any reason for increasing the nginx min replicas?

Reply:
After openshift 101, Jason A/S recommended we try with lower resources and more pods to see how things are working. 
Just so if a node gets evacuated or is faulty or something, we don't have downtime.
Im also curious to see how well this works and if it actually performs better somehow